hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -109,6 +109,30 @@ public void hystrixCommandListFallback() {
     assertThat(command).isNotNull();
     assertThat(command.execute()).containsExactly("fallback");
   }
+  
+  @Test
+  public void hystrixEnhancedFallback() {
+    server.enqueue(new MockResponse().setResponseCode(500));
+
+    TestInterface api = target();
+
+    HystrixCommand<String> command = api.enhanced();
+
+    assertThat(command).isNotNull();
+    assertThat(command.execute()).isEqualTo("fallback with exception");
+  }
+  
+  @Test
+  public void hystrixEnhancedWithArgFallback() {
```

Comment:
let's be consistent. please scrub the word enhanced everywhere as it is a filler term. Choose "WithArgFallback" or something else, and be consistent in both the names of locals and also in test classes and method.